index.php

```diff
@@ -52,7 +52,47 @@
         <h2 class="promo__title">Нужен стафф для катки?</h2>
         <p class="promo__text">На нашем интернет-аукционе ты найдёшь самое эксклюзивное сноубордическое и горнолыжное снаряжение.</p>
         <ul class="promo__list">
-            <!--заполните этот список из массива категорий-->
+            <?php 
+                $cat = ["Ботинки", "Одежда", "Инструменты", "Разное"];
```

Comment:
Рекомендую не сокращать категории до котов :)
```suggestion
                $categories = ["Ботинки", "Одежда", "Инструменты", "Разное"];
```